docker/base/Dockerfile.base

```diff
@@ -24,6 +24,7 @@ RUN apt-get update && \
     ca-certificates \
     openssl \
     libssl-dev \
+    golang \
```

Comment:
I think for now we are not focusing on small proof that can be verified on chain, so I think it'd be better to not install go that increases the image size.

```suggestion
```